pattern-1/artifacts/volumes/persistent-volumes.yaml

```diff
@@ -61,8 +62,9 @@ spec:
   persistentVolumeReclaimPolicy: Delete
   nfs:
   # FIXME: use the right IP
-    server: 10.11.251.230
-    path: "/exports/pattern-1/analytics1"
+  # EXAMLE: /nfs/pattern-1/analytics2
```

Comment:
Shall we change as follows?

#Example Path in NFS server: /exports/pattern-1/anlytics2